src/gui/render.cpp

```diff
@@ -1007,19 +819,37 @@ static void init_render_settings(Section_prop& secprop)
 	string_prop = secprop.Add_string("scaler", deprecated, "none");
 	string_prop->Set_help(
 	        "Software scalers are deprecated in favour of hardware-accelerated options:\n"
-	        "  - If you used the normal2x/3x scalers, set a desired 'windowresolution'\n"
-	        "    instead.\n"
+	        "  - If you used the normal2x/3x scalers, set the desired 'windowresolution'\n"
+	        "    or 'viewport_resolution' instead, or consider using 'integer_scaling'.\n"
 	        "  - If you used an advanced scaler, consider one of the 'glshader'\n"
 	        "    options instead.");
 
 #if C_OPENGL
-	string_prop = secprop.Add_path("glshader", always, "default");
+	string_prop = secprop.Add_path("glshader", always, "crt-auto");
```

Comment:
Previously an empty setting:

```ini
glshader =
```

was behaving like:

```ini
glshader = none
```

Now it prints out a log like:

```
2023-08-23 17:32:46.409 | SDL:OPENGL: Failed to compile shader!
```

and doesn't scale the image at all. I suggest to bring back the previous behavior.

Reply:
Good catch, that wasn't intentional.

Reply:
Fixed.